nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -209,19 +209,17 @@ Use `SELECT * FROM TimeoutsMigration_State` to list all performed migrations. Fo
 
 ### NHibernate persistence
 
- - Delete all the entries from `StagedTimeoutEntity`, that contains the copy of migrated timeouts
- - Delete the table from `MigrationsEntity` that contains a row for every performed migration
+ - Delete all rows from the `StagedTimeoutEntity` table. These are copies of migrated timeouts.
+ - Delete the `MigrationsEntity` table. Each row in this table represents a previously performed migration.
 
 ### Azure Storage persistence
 
-In case all the timeouts in a given timeout table have been migrated the timeout table can be deleted. It is also possible to delete migrated entities in the table only, by deleting all entitites that have an `OwningTimeoutManager` starting with `__hidden__`.
+When all the timeouts in a given timeout table have been migrated, the timeout table may be deleted. It is also possible to delete only migrated entities from a timeout table by deleting all rows with `OwningTimeoutManager` starting with `__hidden__`.
 
-For safety reasons the migration table `timeoutsmigration` is left in the state of the last migration that was run. The table is cleaned automatically for every migration run. Once all migrations are done and the data is no longer used it is advisable to delete
+For safety reasons, the migration table `timeoutsmigration` is left in the state of the last migration that was run. The table is cleaned automatically during every migration run. To reduce storage costs, after all migrations are done it is advisable to delete:
```

Comment:
> For safety reasons...

Can we be more specific? Is it for visibility?

Reply:
I'm guessing this was aimed at tool-recovery/verification scenarios. If we would delete this table at the end of a successful migration, there's no way for the user to verify what was migrated in detail. @danielmarbach @WojcikMike Thoughts?

Reply:
👍 thanks, maybe:

```suggestion
The `timeoutsmigration` table is left in the state of the last migration that was run in case it is required for troubleshooting. The table is cleaned automatically during every migration run. To reduce storage costs, after all migrations are done it is advisable to delete:
```